frontend/src/App.css

```diff
@@ -3,11 +3,7 @@
 }
 
 body{
-    background-color: rgb(248, 246, 246);
-}
-
-.toastContainer{
-
+    background-color: rgba(232, 233, 236, 0.49);
```

Comment:
Ich persönlich würde Farbangabe in Hex umwandeln, aber laufen tut's natürlich auch so. :)